plugins/hint_calls/hint_calls.py

```diff
@@ -371,52 +373,37 @@ def get_custom_viewer_hint(self, view, place):
                 return None
 
             curline = idaapi.get_custom_viewer_curline(view, True)
-            _, x, y = idaapi.get_custom_viewer_place(view, True)
+            
+            # sometimes get_custom_viewer_place() returns [x, y] and sometimes [place_t, x, y].
+            # we want the place_t.
```

Comment:
good catch. i think this explains some error reports i've received, but haven't reproduced.